scripts/protocol-test.sh

```diff
@@ -30,7 +30,7 @@ cargo run --bin pgprototest -- \
     -v || ret=$?
 
 # Kill GlareDB.
-kill "${glaredb_pid}"
+kill "${glaredb_pid}" || true
```

Comment:
failing to kill the process here (if it say, doesn't start up?) means that we don't end up 

we can imagine better implementations of this test: we kill glaredb after we check if the test has failed and report its output, but to do that in shell would involve a lot of `trap` and signal stuff, which I've (delightfully) not had write in a long time, and seems like overkill. I can't imagine a case where we fail to kill glaredb here and where the test would pass and the system would be operable (we start glare as a different user? we could probably SIGKILL here to rule out problems with the signal handler in glare as being a cause for a failed kill), so this seems both safe and was the thing that helped me debug a failed test.